tests/tools/oeseal/host/host.c

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <getopt.h>
+#include <openenclave/host.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/tests.h>
+
+#include "oeseal_u.h"
+
+#define COMMAND_NAME_SEAL "seal"
+#define COMMAND_NAME_UNSEAL "unseal"
+
+#define POLICY_NAME_UNIQUE "unique"
+#define POLICY_NAME_PRODUCT "product"
+
+#define SKIP_RETURN_CODE 2
+
+typedef enum _policy_t
+{
+    POLICY_UNSUPPORTED = 0,
+    POLICY_UNUQUE = 1,
+    POLICY_PORDUCT = 2
+} policy_t;
+
+typedef enum _command_t
+{
+    COMMAND_UNSUPPORTED,
+    COMMAND_SEAL,
+    COMMAND_UNSEAL
+} command_t;
+
+static void _print_helper()
+{
+    printf("Usage: oeseal <command> [<args>]\n"
+           "command: seal, unseal\n"
+           "args:\n"
+           "  -h, --help: Helper message\n"
+           "  -e, --enclave: Enclave binary\n"
+           "  -p, --policy: Sealing policy (product, unique)\n"
```

Comment:
The produce id is hard-coded isn't it? In that case, does the "product" policy help with debugging?

Reply:
Correct. We can remove policy option.

Reply:
removed the policy option.